src/resources/js/components/ListPresentations.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="container">
+    <div v-if="showMessage">
+      <h3>{{ message }}</h3>
+    </div>
+
+    <div v-for="presentation in presentations.data" :key="presentation.id">
+      <router-link
+        :to="{ name : 'presentation', params : { idPresentation : presentation.id} }"
```

Comment:
Why use `'presentation'` as title and not `presentation.title` as per DB model ?

Reply:
here ``presentation`` is the name the route I want to load, but I have set a params name in [472659](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/115/commits/47265981bf2cc727dd23019ad8fb1edde0ca2300) to generate URL like this ``/presentation/name``

Reply:
Commit link seems to be dead, can you check please ?

Reply:
thanks, I corrected it here it is[456b4c](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/115/commits/456b4c09598d5af8177a69c5616178e40388547f)

Reply:
Indeed. Looks good to me